CHANGELOG.md

```diff
@@ -3,3 +3,6 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [Unreleased] - 2018-08-30
```

Comment:
We only put dates into the CHANGELOG right before we do a release, otherwise they get out of date (like has just happened here).

Reply:
removed date in changelog